vip-integrations.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * Plugin Name: VIP Integrations
+ * Description: Plugin loading for VIP plugin integrations.
+ * Author: Automattic
+ * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace Automattic\VIP\Integrations;
+
+if ( ! ( defined( 'VIP_BLOCK_DATA_API_ENABLE' ) && true === constant( 'VIP_BLOCK_DATA_API_ENABLE' ) ) ) {
+	// Do not automatically activate a plugin unless specified.
+	return;
+}
+
+const VIP_BLOCK_API_SUPPORTED_VERSIONS = [
```

Comment:
Instead of supporting multiple versions is this possible to only support one version? I think it will make things easier.

Reply:
Would this mean dropping the `VIP_BLOCK_DATA_API_VERSION` configuration entirely and only using `VIP_BLOCK_DATA_API_ENABLE` as a on-off toggle for the latest version?

Reply:
Yes but that depends whether only supporting the latest version is a valid case for you or not.